### DISCUSSION TITLE
How to use Audit.EntityFramework.Identity for .Net 6

### DISCUSSION THREAD

**Discussion by RickworkGit** [thepirat000/Audit.NET#494] [Q&A]
`Identity` in .Net 6 using razor pages.  I want to audit ASP.NET Identity entities using Audit.EntityFramework.Identity.  But I don't see any document about that. 

I changed my ApplicationDbContext.cs

```
public class ApplicationDbContext : AuditIdentityDbContext<IdentityUser>
{
        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        {
            optionsBuilder.AddInterceptors(new AuditSaveChangesInterceptor());
        }
}
```

I don't see anything write to my Event table.

**Comment by thepirat000** [comment]
Please provide the complete example. i.e. the Audit.Core.Configuration calls

**Reply by RickworkGit** [reply]
I have figured it out already.  Thanks.

**Reply by Jafolayan1** [reply]
Hello @RickworkGit  Can you please share with me what you did ?

**Reply by RickworkGit** [reply]
In AuditConfiguration.cs, 

I added

Audit.Core.Configuration.Setup()
    .UseSqlServer(config => config
        .ConnectionString(connString)
        .Schema("dbo")
        .TableName("Event")
        .IdColumnName("EventId")
        .JsonColumnName("JsonData")
        .LastUpdatedColumnName("LastUpdatedDate")
        .CustomColumn("EventType", ev => ev.EventType)
        .CustomColumn("User", ev => ev.Environment.UserName)
        .CustomColumn("CorrelationId", ev => ev.CustomFields[CorrelationIdField]));

It writes to my [dbo].[Event] table.

**Reply by Jafolayan1** [reply]
Thank you
